pre_commit_dbt/check_exposure_has_models.py

```diff
@@ -0,0 +1,33 @@
+from pre_commit_dbt.utils import get_filenames, get_exposures
+from typing import Sequence
+from pathlib import Path
+from typing import Optional
+import argparse
+from pre_commit_dbt.utils import add_filenames_args
+import logging
+
+def has_models(paths: Sequence[str]):
+    status_code = 0
+    ymls = get_filenames(paths, [".yml", ".yaml"])
+    exposures = get_exposures(list(ymls.values()))
+    missing_model = [exposure.exposure_name for exposure in exposures if exposure.models is None]
+    for exposure in missing_model:
+        status_code = 1
+        print(
+            f"{exposure}: "
+            f"does not have any model",
```

Comment:
And do we really want to print x lines if we have x exposure? Maybe it would be better to display it differently. What do you think @larryandria @benoitgoujon ?

Reply:
I think we need to print x lines if we have x errors. This is the standard for linters to display all errors. Thus, the user can fix all errors at once.